packages/data/src/components/remountOnPropChange/index.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
This folder name should be kebab-case, `remount-on-prop-change`

Reply:
yes, I just noticed it too. This will be fixed in the `@wordpress/compose` PR I'm working on